### ISSUE TITLE
data_loader.load can not give the result

### ISSUE THREAD

**Issue by ZFgan** [microsoft/qlib#422]
When I repeat the following code，

MACD_EXP   = '(EMA($close, 12) - EMA($close, 26))/$close - EMA((EMA($close, 12) - EMA($close, 26))/$close, 9)/$close'
fields = [MACD_EXP]
fiels_names= ['MACD']
labels = ['Ref($close,-2)/Ref($close,-1)-1']
label_names = ['LABEL']
data_handler_config = {
    'feature':(fields,fiels_names),
    'label':(labels,label_names)
}
data_loader = QlibDataLoader(config=data_handler_config)

t1 = time.time()
df = data_loader.load(instruments='csi300',start_time = '2015-01-01',end_time = '2017-12-31')
t2 = time.time()

first time It takes 6s.
second time ，it takes 6s
third time,i can not give the result,It's running all the time.

**Comment by you-n-g** [comment]
@ZFgan 
What error do you get if you interrupt the script by `ctrl+c`

**Comment by you-n-g** [comment]
close this issue due to similar issue in #421
